packages/Search/examples/bvh_driver/bvh_driver.cpp

```diff
@@ -146,9 +146,9 @@ int main_( Teuchos::CommandLineProcessor &clp, int argc, char *argv[] )
         Kokkos::deep_copy( k, k_host );
 
         Kokkos::View<details::Nearest *, DeviceType> nearest_queries(
-            "neatest_queries", n_points );
+            "nearest_queries", n_points );
         Kokkos::parallel_for(
-            "register_nearest_queries",
+            "bvh_driver::main_() register_nearest_queries",
```

Comment:
I think I'm indifferent as long as we have a naming convention that makes it clear where the kernel is located